tests/unit/api/test_metadata.py

```diff
@@ -31,6 +32,13 @@ def test_post_metadata(self, test_client, monkeypatch):
         monkeypatch.setattr(
             "repository_service_tuf_api.metadata.get_task_id", lambda: "123"
         )
+        fake_time = datetime.datetime(2019, 6, 16, 9, 5, 1)
+        fake_datetime = pretend.stub(
+            now=pretend.call_recorder(lambda: fake_time)
+        )
```

Comment:
At the end of the test, you want to add a line testing the `fake_datetime.now` calls.
Check how we assert calls for other call recorders in this function.